### ISSUE TITLE
add classes label?

### ISSUE THREAD

**Issue by qinst64** [sepandhaghighi/pycm#358]
Is it missing or I just did not find a parameter like `lables` to  [sklearn.metrics.confusion_matrix](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html).

for example, I DO know a dataset have labels ['B', 'A', 'C'] and in one sampling I only get data of ['B', 'A'] 
then
  `confusion_matrix(y_true, y_pred, lables=['B', 'A', 'C'])` will calculate a 3*3 confusion matrix and sort in this order in one shot.
however
   how can I achieve this in `pycm.ConfusionMatrix`?

Like add a `classes` parameter to `pycm.ConfusionMatrix`?

**Comment by sepandhaghighi** [comment]
@qinst64 Thanks for your comment 💯 
We will add this option in a future release

**Comment by sadrasabouri** [comment]
Is `class_labels` a good parameter name for it?
`classes` as @qinst64 mentioned may cause some conflicts I wonder.

**Comment by sadrasabouri** [comment]
> Is `class_labels` a good parameter name for it?
> `classes` as @qinst64 mentioned may cause some conflicts I wonder.

Anyway I'll use `classes` and try to avoid conflicts, thanks for your feature request.

**Comment by adrianog** [comment]
`class_labels` is _definitively_ a better name.

**Comment by sadrasabouri** [comment]
> `class_labels` is _definitively_ a better name.

Thank you @adrianog for your comment.
I've pulled a request [here](https://github.com/sepandhaghighi/pycm/pull/373) and we decided to use `classes`. Why do you think `class_labels` will be better?
